cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CreateExecutableTask.java

```diff
@@ -102,11 +106,25 @@ public void execute(Project project) {
 
             // Print warnings for conflicted jars
             if (!jBallerinaBackend.conflictedJars().isEmpty()) {
-                out.println("\twarning: Detected conflicting jar files:");
+                out.println("\twarning: Picking the latest jar since detected conflicting jar files:");
```

Comment:
This doesn't need to change. Two versions of the same library are not considered as conflicts.

Reply:
Reverted the change